### DISCUSSION TITLE
jsonCreate числовое значение в кавычках как строка

### DISCUSSION THREAD

**Discussion by nikidjan** [totumonline/totum-ru-issues-and-discussions#448] [Как это сделать?] [answered]
мне нужно json отправить POST запросом, а jsonCreate (а так же и rowCreate) числа оборачивает в кавычки почему то
например:
`json: jsonCreate(field: "description" = "Meeting with clients"; field: "billable" = false; field: "tags" = $#lc; field: "duration" = 1200; field: "start" = "2023-10-01T11:58:58.000Z";)`
на выходе будет:
`{"description":"Meeting with clients","billable":false,"tags":[],"duration":"1200","start":"2023-10-01T11:58:58.000Z"}`
1200 в кавычках 
--
версия totum 4.9.52.3-4.0

**Comment by erpic-ru** [comment, marked as answer]
В тотуме все строка, поэтому, если сильно нужно получить число в json, то это делается при помощи функции numTransform: 
https://ru.docs.totum.online/functions#fn-numTransform

**Reply by nikidjan** [reply]
огромное спасибо! я так и думал, вроде поискал по документации но видимо пропустил
